### ISSUE TITLE
Failed to fetch

### ISSUE THREAD

**Issue by alexdhika** [WordOps/WordOps#175]
E: Failed to fetch http://download.opensuse.org/repositories/home:/virtubox:/WordOps/xUbuntu_18.04/InRelease  403  Forbidden [IP: 195.135.221.134 80]
E: The repository 'http://download.opensuse.org/repositories/home:/virtubox:/WordOps/xUbuntu_18.04  InRelease' is no longer signed.

**Comment by VirtuBox** [comment]
Hello @alexdhika,
can you provide more information about the issue ?

**Comment by risturiz** [comment]
Hi, actually im having invalid signature with last version:
The following signatures were invalid: EXPKEYSIG 3050AC3CD2AE6F03 home:rtCamp OBS Project <home:rtCamp@build.opensuse.org>

**Comment by VirtuBox** [comment]
Hello @risturiz,
this is the old EasyEngine Nginx repository. You can remove it from `/etc/apt/sources.list.d/ee-repo.list`

**Comment by risturiz** [comment]
> Hello @risturiz,
> this is the old EasyEngine Nginx repository. You can remove it from `/etc/apt/sources.list.d/ee-repo.list`

Actually i dont have ee-repo.list ... The old repo is inside "sources.list.d/wo-repo.list":

```
deb [arch=amd64,ppc64el] http://sfo1.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu bionic main
deb http://download.opensuse.org/repositories/home:/rtCamp:/EasyEngine/xUbuntu_18.04/ /
deb http://download.opensuse.org/repositories/home:/virtubox:/WordOps/xUbuntu_18.04/ /
```

**Comment by VirtuBox** [comment]
Issue has been fixed in the last release (v3.9.9.4)

Please update WordOps with the command `wo update`
